.github/workflows/ci.yml

```diff
@@ -61,6 +61,21 @@ jobs:
       - name: test doc no-default-features
         run: cargo test -p prost-build -p prost-derive -p prost-types --doc --no-default-features
 
+  msrv:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: install protoc
+        uses: taiki-e/install-action@v2
+        with:
+          tool: protoc@${{ env.PROTOC_VERSION }}
+      - uses: taiki-e/install-action@cargo-hack
+      - uses: Swatinem/rust-cache@v2
+      - name: check with no-default-features
+        run: cargo hack --rust-version --no-private --no-dev-deps check --workspace --no-default-features
```

Comment:
What is the added value of `--no-dev-deps`? I read the docs, but I don't understand what it brings to a MSRV check.

Reply:
`cargo-hack` removes `dev-dependencies` temporarily when being performed with `--no-dev-deps` option. As dependencies are affected by `dev-dependencies` and they might affect MSRV checking, this option is used to remove the effects.

Reply:
That makes sense, thanks